paddle/operators/pool_op.cc

```diff
@@ -80,34 +80,31 @@ Pool2dOpMaker::Pool2dOpMaker(framework::OpProto *proto,
             "the number of channels, H and W is the height and "
             "width of feature.");
 
-  AddAttr<std::string>("pooling_type",
-                       "Pooling_type of pooling operator."
+  AddAttr<std::string>("poolingType",
+                       "(string), poolingType of pooling operator."
```

Comment:
(string), pooling type, can be "max" for max-pooling and "avg" for average-pooling.

Reply:
Done